src/components/Dashboard.vue

```diff
@@ -529,40 +527,64 @@ export default {
     },
 
     endOnboarding(onboardingData) {
+      console.log(onboardingData);
       const user = this.parseUserData(onboardingData.userData, onboardingData.name);
 
       this.user = user;
       this.loaded = true;
 
       const docRef = this.getDocRef();
-
       const data = {
         name: onboardingData.name,
         userData: onboardingData.userData,
         semesters: this.firebaseSems,
         subjectColors: this.subjectColors
       };
-
       // set the new name and userData, along with either an empty list of semesters or preserve the old list
       docRef.set(data);
-
       this.cancelOnboarding();
       this.updateRequirementsMenu();
     },
 
     cancelOnboarding() {
       this.isOnboarding = false;
     },
-
     parseUserData(data, name) {
       const user = {
         // TODO: take into account multiple majors and colleges
         college: data.colleges[0].acronym,
         collegeFN: data.colleges[0].fullName,
         firstName: name.firstName,
         middleName: name.middleName,
-        lastName: name.lastName
+        lastName: name.lastName,
+        exam: [],
+        transferCourse: [],
+        tookSwim: data.tookSwim
       };
+      const transferClasses = [];
+      if ('exam' in data && data.exam.length > 0) {
+        const exams = [];
+        data.exam.forEach(exam => {
+          exams.push(exam);
+          if ('equivCourse' in exam) {
+            transferClasses.push(exam.equivCourse[0]);
+          }
+        });
+        user.exam = exams;
+      }
+      if ('class' in data && data.class.length > 0) {
+        const classes = [];
+        for (const course of data.class) {
+          classes.push(course);
+          const courseInfo = this.createCourse(course.course);
+          transferClasses.push(courseInfo);
+          // ; // TODO for user to pick which classes it goes for
+          // console.log('E', crs);
```

Comment:
What is this for? Letting a user pick what req a class goes for? Maybe clean up this comment and remove the console statement